example/CMakeLists.txt

```diff
@@ -19,15 +19,35 @@
 # SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 ###############################################################################
 
+include(OmrAssert)
+
+omr_assert(TEST OMR_EXAMPLE)
+
+add_library(omr_example_base INTERFACE)
+
+target_include_directories(omr_example_base
+	INTERFACE
+		glue
+)
+
 add_executable(example
 	main.cpp
 )
 
 target_link_libraries(example
-	PRIVATE
-	omrgc
+	omr_main_function
+	pugixml
+	omrtestutil
```

Comment:
Is omrtestutil still required now that you moved omr_main_function out into a separate library?

Reply:
Is the thumbs up a yes that it is still required or a yes I will remove it since it isn't required anymore?

Reply:
It needs to be removed. I'll get to it today.

Reply:
done.